### PR TITLE
Added spacing between list items

### LINKED ISSUE #7400: Add spacing between list items
Lists with more than a few words per item are difficult to read with no extra spacing between the individual points (both for numbered and unnumbered lists). I'm not sure how much spacing would be necessary (perhaps the same as between paragraphs?)

screenshot_2017-03-27_19-51-09

### PR DESCRIPTION
**Pull request by Waithamai**
fixes #7400

### CODE REVIEW

File: app/assets/stylesheets/markdown-content.scss [hunk 1]
```diff
@@ -3,8 +3,9 @@
   p:last-child { margin-bottom: 0; }
 
   ul, ol {
-    margin-top:0.8em;
-    margin-bottom:0.8em;
+    margin-top: 0.8em;
```
Comment (diaspora-code-review) [inline app/assets/stylesheets/markdown-content.scss]: - `0.8` should be written without a leading zero as `.8`
- Properties should be ordered margin-bottom, margin-top

File: app/assets/stylesheets/markdown-content.scss [hunk 2]
```diff
@@ -3,8 +3,9 @@
...
+    margin-bottom: 0.8em;
```
Comment (diaspora-code-review) [inline app/assets/stylesheets/markdown-content.scss]: `0.8` should be written without a leading zero as `.8`

File: app/assets/stylesheets/markdown-content.scss [hunk 3]
```diff
@@ -3,8 +3,9 @@
...
+    li { margin-bottom: 0.5em; }
```
Comment (diaspora-code-review) [inline app/assets/stylesheets/markdown-content.scss]: `0.5` should be written without a leading zero as `.5`

**Review by skyschub — APPROVED** [review]
Tests are green and both @svbergerem and @SuperTux88 thumb'ed up the `.5em` in the issue, so I guess this is fine to merge.

**Comment by denschub** [comment]
Merged as 38450eacea52b241b92b9b8a4d2b0ce7c186e6fe.

Thank you very much and congrats to your first PR. 🍪🍰🍪🎉🍪🍰🍪

**Comment by svbergerem** [comment]
Looks much better. :)

List item spacing comparison

**Pull request closed without merging**